index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title"> survey form for microverse </h1>
+<P id="description">This is a test for the trials</P> 

```

Comment:
♻️ Great description for the page! However, please remember that to make it better, the p tag should be in lowercase to follow good practices. Lowercase HTML element names are easier to read and understand, which makes it simpler for developers to debug code.